esmvaltool/cmorizers/data/formatters/datasets/gpcp_sg.py

```diff
@@ -13,6 +13,11 @@
 Download and processing instructions
     Download the file precip.mon.mean.nc
     wget https://downloads.psl.noaa.gov/Datasets/gpcp/precip.mon.mean.nc
+    or
+    use the automatic download script shipped by ESMValTool
+    esmvaltool data download GPCP-SG
+    or
+    esmvaltool data prepare GPCP-SG
 """
```

Comment:
These changes are for the dataset GPCP-SG. Did you change this on purpose?

Reply:
Okay, I deleted the lines. I added them because when I tested the download and preparation I had to look those commands up and woudl have been happy about the reminder.